### DISCUSSION TITLE
Enhancement: Is there a plan to support GraphRAG

### DISCUSSION THREAD

**Discussion by Passerby1011** [danny-avila/LibreChat#3366] [Q&A]
### What features would you like to see added?
Add support for GraphRAG, GraphRAG is more effective than regular RAG, the current RAG lacks a complete understanding of the document at the time, and can only serve as a search tool.

### More details
https://github.com/microsoft/graphrag

### Which components are impacted by your request?
Plugins

### Code of Conduct
* [x]  I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Improved RAG overall is planned

**Comment by tiefighter** [comment]
LightRAG offer superior results over simple RAG.
